pkgs/applications/virtualization/docker/default.nix

```diff
@@ -44,6 +44,13 @@ rec {
         rev = containerdRev;
         sha256 = containerdSha256;
       };
+
+      # This should go into the containerd derivation once 1.0.0 is out
+      preBuild = ''
+        mkdir $(pwd)/vendor/src
```

Comment:
This breaks `docker_17_06` which already has `vendor/src`: https://github.com/containerd/containerd/tree/6e23458c129b551d5c9871e5174f6b1b7f6d1170/vendor/src

Reply:
https://github.com/NixOS/nixpkgs/pull/29888

Reply:
Oh s**t, yes... Thank you for fixing it!